build/stage-build.yml

```diff
@@ -39,7 +46,7 @@ jobs:
         iOS:
           ApplicationConfiguration: Release_iOS
           ApplicationPlatform: Any CPU
-          ArtifactName: $(iOSArtifactName)    
+          ArtifactName: $(iOSArtifactName)_${{ parameters.applicationEnvironment }}
```

Comment:
Was just wondering, what's the _$ append before this parameters variables, concatenation ?

Reply:
The artifact name will be "iOS_Production" or "iOS_Staging"